autosklearn/automl.py

```diff
@@ -926,14 +922,18 @@ def _check_X(self, X):
 
     def _check_y(self, y):
         y = sklearn.utils.check_array(y, ensure_2d=False)
-
         y = np.atleast_1d(y)
+
+        if y.ndim == 1:
+            return y
         if y.ndim == 2 and y.shape[1] == 1:
             warnings.warn("A column-vector y was passed when a 1d array was"
                           " expected. Will change shape via np.ravel().",
                           sklearn.utils.DataConversionWarning, stacklevel=2)
             y = np.ravel(y)
+            return y
 
+        self._n_outputs = y.shape[1]
```

Comment:
Could you please briefly describe why this is necessary here?

Reply:
I think this is redundant. Did not find it out. Thanks!

Reply:
Just found out this should be modified and moved to **AutoMLRegressor** part. Under the circumstance when using cross validation, the refit function check if self._n_output matches targets' _n_output. 
self._n_output was only initialized as 1 and I did not update it there in Regressor function, now solved.